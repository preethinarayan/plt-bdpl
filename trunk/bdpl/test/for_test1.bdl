byte i;
for(i=65;i<91;i=i+1) printstring(i);
for(i=65;i<91;i=i+1) print(i);