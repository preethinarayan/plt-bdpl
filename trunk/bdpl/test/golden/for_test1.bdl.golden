A0x41B0x42C0x43D0x44E0x45F0x46G0x47H0x48I0x49J0x4aK0x4bL0x4cM0x4dN0x4eO0x4fP0x50Q0x51R0x52S0x53T0x54U0x55V0x56W0x57X0x58Y0x59Z0x5a